src/components/dialogs/connectivity/connectivity-form.jsx

```diff
@@ -117,14 +119,15 @@ export const ConnectivityForm = ({
     useEffect(() => {
         const currentBusOrBusbarSection = getValues(`${id}.${BUS_OR_BUSBAR_SECTION}`);
         if (
+            !isEquipmentModification &&
```

Comment:
Since it is a pre-condition, in my opinion we can do as following
```ts
        if (isEquipmentModification) {
            return;
        } 
```
before 
`const currentBusOrBusbarSection = getValues(`${id}.${BUS_OR_BUSBAR_SECTION}`);`